modules/nw-sriov-installing-operator.adoc

```diff
@@ -81,52 +78,83 @@ metadata:
   name: sriov-network-operator-subscription
   namespace: openshift-sriov-network-operator
 spec:
-  channel: <channel> <1>
+  channel: "<channel>" <1>
   name: sriov-network-operator
   source: redhat-operators <2>
   sourceNamespace: openshift-marketplace
 ----
-<1> Specify the value from you obtained in the previous step for the `.status.defaultChannel` parameter.
-<2> You must specify the `redhat-operators` value.
+<1> Specify the value of the `status.defaultChannel` field.
+<2> The value of `spec.source` must be `redhat-operators`.
 
-.. Create the Subscription object by running the following command:
+. To create the Subscription object, enter the following command:
 +
 ----
 $ oc create -f sriov-sub.yaml
 ----
 
-.. Change to the `openshift-sriov-network-operator` project:
+. To verify that the Operator is installed, enter the following command:
 +
 ----
-$ oc project openshift-sriov-network-operator
-
-Now using project "openshift-sriov-network-operator"
+$ oc get csv -n openshift-operator-lifecycle-manager
+----
++
+If the installation is successful, the command returns output similar to the following:
++
+----
+NAME            DISPLAY          VERSION   REPLACES   PHASE
+packageserver   Package Server   0.14.2               Succeeded
 ----
 
 [id="install-operator-web-console_{context}"]
-== Installing the Operator using the web console
+== Web console: Installing the SR-IOV Network Operator
 
 As a cluster administrator, you can install the Operator using the web console.
 
 [NOTE]
 ====
-You have to create the Namespace CR and OperatorGroup CR as mentioned
-in the previous section.
+You must create both the Namespace CR and OperatorGroup CR by using the CLI.
 ====
 
+.Prerequisites
+
+* A cluster installed on bare-metal hardware with nodes that have hardware that supports SR-IOV.
+* The {product-title} Command-line Interface (CLI), commonly known as `oc`.
+* An account with `cluster-admin` privileges.
+
 .Procedure
 
+. To create a namespace for the SR-IOV Network Operator, complete the following steps:
```

Comment:
This feels less and less like content that should be in a single module. I'd consider rechunking the procedures to for better reuse instead of repeating this section.

Reply:
@kalexand-rh, I'll see if a Namespace CR can be created in the Web console, so the procedure is at least consistent.